src/urllib3/util/request.py

```diff
@@ -80,11 +79,13 @@ def make_headers(
         headers["connection"] = "keep-alive"
 
     if basic_auth:
-        headers["authorization"] = "Basic " + b64encode(b(basic_auth)).decode("utf-8")
+        headers["authorization"] = "Basic " + b64encode(
```

Comment:
`b64encode` can only return ASCII so we can do `b64encode(x.encode("latin-1")).decode()`. Also we can use an `f-string here instead of string concat